Gemfile.base

```diff
@@ -75,12 +75,12 @@ gem 'htmlentities', '~>4.3', '>= 4.3.4'
 gem 'inherited_resources', '~> 1.7.2'
 gem 'json', '~> 1.8.1'
 
-case ENV.fetch('DATABASE', 'mysql')
-when 'postgresql'
-  gem 'pg'
-when 'mysql'
-  gem 'mysql2', '~>0.3.11'
+gem 'mysql2', '~>0.3.11'
```

Comment:
Why do we install the mysql gem always now even when we use the postgresql DB?

Reply:
Thats a requirement of https://github.com/pat/thinking-sphinx. In the [Installation notes](https://github.com/pat/thinking-sphinx#installation), they say:
> The MySQL gems mentioned are required for connecting to Sphinx, so please include it even when you’re using PostgreSQL for your database.

Reply:
Right it would be good to update the INSTALL.md so that everything is installed once and for all in development too.